src/main/java/mods/eln/Eln.java

```diff
@@ -6616,18 +6911,83 @@ private void recipeTreeResinAndRubber() {
     }
 
     private void recipeRawCable() {
-        addRecipe(findItemStack("Copper Cable", 6),
+        addRecipe(findItemStack("Copper Cable", 12),
             "III",
             'I', "ingotCopper");
 
-        addRecipe(findItemStack("Iron Cable", 6),
+        addRecipe(findItemStack("Iron Cable", 12),
             "III",
             'I', new ItemStack(Items.iron_ingot));
 
         addRecipe(findItemStack("Tungsten Cable", 6),
             "III",
             'I', dictTungstenIngot);
+        addRecipe(findItemStack("T1 Transmission Cable", 6),
+            "III",
+            'I', firstExistingOre("ingotSteel", "Arc Metal Ingot"));
+        addRecipe(findItemStack("T2 Transmission Cable", 6),
+            "III",
+            'I', firstExistingOre("ingotAluminium","ingotAluminum","Arc Clay Ingot"));
 
+    }
+	private void recipeGraphite() {
+		addRecipe(findItemStack("Creative Cable", 1),
+            "I",
+			"S",
+            'S', findItemStack("unreleasedium"),
+			'I', findItemStack("Synthetic Diamond"));
+		addRecipe(findItemStack("Arc Clay Block", 1),
+            "III",
+			"III",
+			"III",
+            'I', findItemStack("Arc Clay Ingot"));
+		addRecipe(findItemStack("Arc Clay Ingot", 9),
+            "I",
+            'I', findItemStack("Arc Clay Block"));
+		addRecipe(findItemStack("Graphite Rod", 2),
+            "I",
+            'I', findItemStack("2x Graphite Rods"));
+		addRecipe(findItemStack("Graphite Rod", 3),
+            "I",
+            'I', findItemStack("3x Graphite Rods"));
+		addRecipe(findItemStack("Graphite Rod", 4),
+            "I",
+            'I', findItemStack("4x Graphite Rods"));
+        addShapelessRecipe(
+            findItemStack("2x Graphite Rods"),
+			findItemStack("Graphite Rod"),
+			findItemStack("Graphite Rod"));
+		addShapelessRecipe(
+            findItemStack("3x Graphite Rods"),
+			findItemStack("Graphite Rod"),
+			findItemStack("Graphite Rod"),
+			findItemStack("Graphite Rod"));
+		addShapelessRecipe(
+            findItemStack("3x Graphite Rods"),
+			findItemStack("Graphite Rod"),
+			findItemStack("2x Graphite Rods"));
+		addShapelessRecipe(
+            findItemStack("4x Graphite Rods"),
+			findItemStack("Graphite Rod"),
+			findItemStack("Graphite Rod"),
+			findItemStack("Graphite Rod"),
+			findItemStack("Graphite Rod"));
+		addShapelessRecipe(
+            findItemStack("4x Graphite Rods"),
+			findItemStack("2x Graphite Rods"),
+			findItemStack("Graphite Rod"),
+			findItemStack("Graphite Rod"));
+		addShapelessRecipe(
+            findItemStack("4x Graphite Rods"),
+			findItemStack("2x Graphite Rods"),
+			findItemStack("2x Graphite Rods"));
+		addShapelessRecipe(
+            findItemStack("4x Graphite Rods"),
+			findItemStack("3x Graphite Rods"),
+			findItemStack("Graphite Rod"));
+		addShapelessRecipe(
+            new ItemStack(Items.diamond, 2),
```

Comment:
If the intent is to use two diamonds to craft a Synthetic Diamond in a crafting table, it's not working.

Reply:
Other way around. One synth converts into two diamonds. It used to be 1:1 but I couldn't justify keeping it that way due to the high costs involved in making a synth diamond, and how much more valuable the synth diamond was over the lesser version.